pkgs/applications/blockchains/chia-plotter/default.nix

```diff
@@ -9,13 +9,13 @@
 
 stdenv.mkDerivation {
   pname = "chia-plotter";
-  version = "unstable-2021-07-12";
+  version = "1.1.7";
 
   src = fetchFromGitHub {
     owner = "madMAx43v3r";
     repo = "chia-plotter";
-    rev = "974d6e5f1440f68c48492122ca33828a98864dfc";
-    sha256 = "0dliswvqmi3wq9w8jp0sb0z74n5k37608sig6r60z206g2bwhjja";
+    rev = "18cad340858f0dbcc8dafd0bda1ce1af0fe58c65";
```

Comment:
Checks: https://github.com/madMAx43v3r/chia-plotter/commit/18cad340858f0dbcc8dafd0bda1ce1af0fe58c65

Upstream doesn't seem to provide releases or tags.